### ISSUE TITLE
Celery/worker config

### ISSUE THREAD

**Issue by rolanddb** [gsemet/charts#9]
Continuing the discussion from the (already closed) PR gsemet/charts#6

We now have a values.yaml that looks like this

```workers:
  serviceAccountName: "default"
  replicas: 1

  celery:
    # instances per worker
    instances: 1
```

If I understand correctly `workers.replicas` should configure the amount of Pods in the airflow-worker StatefulSet. 
And `workers.celery.instances` should configure the number of task instances for Celery. This can be done via celeryd_concurrency in airflow.cfg (https://github.com/puckel/docker-airflow/blob/master/config/airflow.cfg#L241).

@gsemet Do you agree?

In the current implementation of this Chart, I confused a setting. So currently the amount of replicas is set by `.Values.workers.celery.instances`, and `.Values.workers.replicas` does nothing.

**Comment by gsemet** [comment]
Yes I guess airflow handle this that way. We may also want to expose CPU and pem per worker.
